Document perfect PRF/PRP assumption (#32)
cfrg/draft-irtf-cfrg-aead-limits: draft-irtf-cfrg-aead-limits.md

# Security Considerations {#sec-considerations}

The different analyses of AEAD functions that this work is based upon generally
assume that the underlying primitives are ideal.  For example, that the

> pseudorandom function (PRF) or pseudorandom permutation (PRP) the AEAD builds
> upon produces output that is indistinguishable from random.  Thus, the advantage

Reviewer: ```suggestion
pseudorandom function (PRF) or pseudorandom permutation (PRP) the AEAD builds
upon is indistinguishable from a truly random function.  Thus, the advantage
```
